packages/components/src/Autocomplete/Autocomplete.tsx

```diff
@@ -100,14 +104,13 @@ export function Autocomplete({
       <InputText
         autocomplete={false}
         size={size}
-        invalid={invalid}
```

Comment:
This is weird. maybe this is doing something? doesn't seem to highlight red anymore

![image](https://github.com/GetJobber/atlantis/assets/15986172/d1908e86-83e4-4ef6-a0b0-b8beb0559930)

compared to master

<img width="386" alt="image" src="https://github.com/GetJobber/atlantis/assets/15986172/da121335-8444-424b-b1a6-5de2ee755051">

Reply:
Hmm interesting - I'll check it out!

Reply:
Not seeing the same behaviour on the [storybook preview link](https://feat-autocomplete-props.atlantis.pages.dev/): 

<img width="1632" alt="storybook" src="https://github.com/GetJobber/atlantis/assets/6474230/1aa2d998-e12a-41be-964d-2e95dc9d6d80">

Validations and error highlighting appear to be functioning as expected?

Reply:
Oh dang, you right!. probs just my local being a silly goose!